static/homepage/install.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Install Page Shot</title>
+    <style type="text/css">
+html {
+  font-family: sans-serif;
+}
+    </style>
+  </head>
+  <body>
+    <center>
+      <img src="/static/img/pageshot.svg" height="256" width="256" />
+      <h1>Page Shot</h1>
+      <a href="https://testpilot.firefox.com/files/pageshot/pageshot-0.1.201608231911.xpi">Install</a>
```

Comment:
Personal curiosity, but would it be possible to host the add-on on the local PageShot server?
That way we'd be able to go to dev/stage/prod servers and get the appropriate latest add-on that matches the server.